### ISSUE TITLE
Websocket and server access error in dev

### ISSUE THREAD

**Issue by jgwerner** [3Blades/3blades#189]
There was an error when starting and attempting to launch servers in **dev** with standard data science notebook image. The error appears to have disapeared with latest dev deployment. Error appeared with servers created with existing and new projects:

```
index.js:13 WebSocket connection to 'wss://dev-api.3blades.ai/v1/jgwerner/projects/6dc26de5-f744-41b3-928f-7691710a8efd/servers/1065e941-5657-4529-8fc6-feb9d79a4313/status/' failed: Error during WebSocket handshake: Unexpected response code: 404
index.js:67 WebSocket connection to 'wss://dev-api.3blades.ai/v1/jgwerner/projects/6dc26de5-f744-41b3-928f-7691710a8efd/servers/1065e941-5657-4529-8fc6-feb9d79a4313/logs/' failed: Error during WebSocket handshake: Unexpected response code: 404
```

**Comment by jgwerner** [comment]
@johngriebel @roachmd will leave open for you guys to sign off. Will also re-test on home computer where the error was occurring.

**Comment by johngriebel** [comment]
@jgwerner @roachmd I'm going to close this since we solved it as part of #192
